### PR TITLE
Refresh docs

### LINKED ISSUE #245: Updates to installation instructions
On the topic of #210 I think we should add two additional things to the installation documentation.

* We should add instructions for `conda` users in the "Installation via pip" section. I'm guessing most outside collaborators using OPSO will be using `conda`, and for quite a few of them, it will be their first foray into virtual environments.
* We already have Jupyter instructions in the "poetry" section of the Installation instructions. It would be helpful to add a short section about Jupyter to the "pip" section of these instructions as well. This section could explicitly describe how to start a JupyterLab instance and make a note about how to download one of the tutorial notebooks and use it in JupyterLab.

I'll make these additions if there are no objections.

### PR DESCRIPTION
**Pull request by rhine3**
Updates documentation for OpenSoundscape in several ways:
* Adds instructions to RTD for OPSO on Windows (supersedes #244, addresses #210)
* Reorganizes installation instructions on RTD and removes installation instructions from README (closes #245)
* Reorganizes API documentation into coherent groups
* Refreshes Jupyter Notebooks, removing deprecated functionality
* Updates docstrings in many functions to "Args/Returns" formatting, enabling autodoc to format these nicely
* Gives the website a fresh new color scheme :-)

Made two minor changes to Black checks as well
* [Removed Python 3.7 precommit requirement](https://github.com/kitzeslab/opensoundscape/pull/259/commits/bf93b56517524e145b8c8e76e10766f0dfd14493) because it prevented me from using the precommit on a Python 3.8 environment
* [Made failing CIs more verbose](https://github.com/kitzeslab/opensoundscape/pull/259/commits/a541a97de6d0d7334f4abc26f0d8579a822dfff4) which somehow fixed a problem where black checks failed on GitHub but passed locally

### CODE REVIEW

**Comment by rhine3** [comment]
Still to address before merging this:
* [ ] #252 add links 
* [ ] #245 delete deprecated branch and close

File: .github/workflows/poetry.yml [hunk 1]
```diff
@@ -29,4 +29,4 @@ jobs:
       - name: Poetry run pytest
         run: ~/.poetry/bin/poetry run pytest
       - name: Poetry run black check
-        run: ~/.poetry/bin/poetry run black . --check
+        run: ~/.poetry/bin/poetry run black . --check --diff
```
Comment (sammlapp) [inline .github/workflows/poetry.yml:32]: what does this do?

File: README.md [hunk 2]
```diff
@@ -17,109 +17,6 @@ OpenSoundscape is currently in active development. If you find a bug, please sub
...
-  * To automatically handle this, `poetry run pre-commit install`
-* Additional libraries to be installed should be installed with `poetry add`, but
-  in most cases contributors should not add libraries.
-
-### Contributing to documentation
-
-Build the documentation using either poetry or sphinx-build
-- With poetry: `poetry run build_docs`
-- With sphinx-build: `sphinx-build doc doc/_build`
+OpenSoundscape should be run on Python 3.7 and can be installed via pip: `pip opensoundscape==0.4.6`. For more detailed instructions and tutorials, see the documentation.
```
Comment (sammlapp) [inline README.md:22]: maybe "has been tested on Python 3.7" 
I think 3.8 and maybe even 3.9 are probably fine if someone has a reason to use them

**Review by sammlapp — APPROVED** [review]
I didn't inspect the raven stuff closely. Let me know if you want me to take a closer look at it or try it. Added a couple small comments

File: .github/workflows/poetry.yml (hunk 1 above)
Comment (rhine3) [reply .github/workflows/poetry.yml:32]: During CI checks, this flag causes the check to print out the formatting changes that would happen for any unformatted files:

Here's an example
https://github.com/kitzeslab/opensoundscape/runs/1989043264

File: README.md (hunk 2 above)
Comment (rhine3) [reply README.md]: Good point, made this update

**Pull request merged**